starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,135 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:light:300,600');
+
+/*I have spent too much time looking for the font, I know it's a sans serif typeface
+looks like Roboto, Open Sans, and a lot like Proxima Nova Light, $$$.*/
+
+/*Sherwin, don't forget to change all of the px values to em values
+It has better scaling abilities*/
+
+/*I am going to test out this @media scalling css I found online...cause some people with retina
+displays are going to see this site and it needs to be optimized for such. Only problem I am having right now
+is with the positioning of the imgparty class. */
+
+@media screen and (orientation: portrait) {
+  body { max-width: 100%; }
+}
+
+@media screen and (orientation: landscape) {
+  body { max-width: 100%; }
+}
+
```

Comment:
Good to see you looking ahead/looking at outside resources.  We'll go over these media queries in the near future.